generators/chipyard/src/main/scala/IOBinders.scala

```diff
@@ -182,6 +183,20 @@ object AddIOCells {
     port.suggestName("serial")
     (port, ios)
   }
+
+  def axi4(io: Seq[AXI4Bundle], node: AXI4SlaveNode): Seq[(AXI4Bundle, AXI4EdgeParameters, Seq[IOCell])] = {
```

Comment:
Why does this return the edge? It doesn't seem to be used.

Reply:
it is used to construct both the `SimAXIMem` and the `SimDRAM`.

Reply:
Oh I see it now. Is this `edge` just the bundle parameters? I think this is fine, it's just surprising to me to see a diplomacy edge getting passed around here.

Reply:
Its slightly more than the bundle parameters, but `SimAXIMem` takes in the whole edge while `SimDRAM` only takes in the bundle params.